Workbooks/CosmosDb/Mongo/Highest RU consumption.workbook

```diff
@@ -0,0 +1,1254 @@
+        }
+      ],
+      "name": "query - 14"
+    },
+    {
+      "type": 3,
+      "content": {
+        "version": "KqlItem/1.0",
+        "query": "//AzureDiagnostics\r\n//| where ResourceProvider==\"MICROSOFT.DOCUMENTDB\" and Category == 'PartitionKeyStatistics'\r\n\r\nCDBMongoRequests\r\n| where _ResourceId == tolower('{ResourceId}')\r\n| where DatabaseName == '{Database}' and CollectionName == '{Container}'\r\n| summarize throttledOperations = dcountif(ActivityId, ErrorCode == 50), totalOperations = dcount(ActivityId), totalConsumedRUPerMinute = sum(todouble(RequestCharge)) by OperationName, CollectionName, bin(TimeGenerated, 1min)\r\n| extend averageRUPerOperation = 1.0 * totalConsumedRUPerMinute / totalOperations \r\n| extend fractionOf429s = 1.0 * throttledOperations / totalOperations\r\n| order by fractionOf429s desc\r\n\r\n\r\n\r\n\r\n\r\n//|",
+        "size": 0,
+        "queryType": 0,
+        "resourceType": "microsoft.operationalinsights/workspaces",
+        "crossComponentResources": [
+        ],
+        "gridSettings": {
+          "rowLimit": 500
+        }
+      },
+      "conditionalVisibilities": [
+        {
+          "parameterName": "IsThrottled",
+          "comparison": "isEqualTo",
+          "value": "1"
+        },
+        {
+          "parameterName": "Container",
+          "comparison": "isNotEqualTo"
+        }
+      ],
+      "name": "query - 14 - Copy"
+    },
+    {
+      "type": 1,
+      "content": {
+        "json": "# Use the Azure Cosmos DB capacity planner\r\nYou can use the Azure Cosmos DB capacity planner to understand what is the best provisioned throughput based on your workload (volume and type of operations and size of documents). You can customize further the calculations by providing sample data to get a more accurate estimation.\r\n\r\n## 50s on create, replace, or upsert document requests\r\nBy default, in the SQL API, all properties are indexed by default. Tune the <a href=\"https://docs.microsoft.com/azure/cosmos-db/index-policy\">indexing policy</a> to only index the properties needed. This will lower the Request Units required per create document operation, which will reduce the likelihood of seeing 50s or allow you to achieve higher operations per second for the same amount of provisioned RU/s.\r\n\r\n## 50s on query document requests\r\nFollow the guidance to <a href=\"https://docs.microsoft.com/azure/cosmos-db/sql/troubleshoot-query-performance#querys-ru-charge-is-too-high\">troubleshoot queries with high RU charge</a>\r\n\r\n## 50s on execute stored procedures\r\n<a href=\"https://docs.microsoft.com/azure/cosmos-db/sql/stored-procedures-triggers-udfs\">Stored procedures</a> are intended for operations that require write transactions across a partition key value. It is not recommended to use stored procedures for a large number of read or query operations. For best performance, these read or query operations should be done on the client-side, using the Cosmos SDKs."
```

Comment:
This is describing the SQL API. Seems out of place in this workbook. Should we remove this or edit this for the Mongo API?

Reply:
I'd suggest removing this section to unblock us.